<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>have_select</class>
 <widget class="QDialog" name="have_select">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>559</width>
    <height>359</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>已选界面</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>ok</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
